woocommerce/woocommerce
# move action scheduler to external via composer

### All Submissions:


### Changes proposed in this Pull Request:

This PR removes the library copy of Action Scheduler and adds it as an external through composer. The version of AS used in this PR is the same version as was in the `libraries` folder (2.2.5). This will ensure that any issues found in testing are related to the change in inclusion. AS can be updated to 3.0.X in a follow up PR.

### How to test the changes in this Pull Request:

1. Run `composer install`
2. Add a webhook to `order.create`
3. Create an order
4. Verify that the pending webhook action is created in Tools -> Scheduled Actions
5. Run `wp action-scheduler run`
6. Verify the action was completed and the webhook processed to the destination.

### Other information:




### Changelog entry
